_data/sites.json

```
@@ -17287,6 +17287,18 @@
         ]
     },
 
+    {
+        {
+            "name": "PrestaShop",
+            "url": "https://accounts.distribution.prestashop.net/settings/delete-account",
+            "difficulty": "easy",
+            "notes": "Fill out the form.",
+            "domains": [
+                "prestashop.com"
+            ]
+        }
+    },
```

You added one pair of braces {} too much.

```suggestion
    {
        "name": "PrestaShop",
        "url": "https://accounts.distribution.prestashop.net/settings/delete-account",
        "difficulty": "easy",
        "notes": "Fill out the form.",
        "domains": [
            "prestashop.com"
        ]
    },
```

---

Oops 😅